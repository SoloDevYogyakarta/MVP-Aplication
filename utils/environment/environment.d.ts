/// <reference types="node" />
export declare const environment: NodeJS.ProcessEnv;
