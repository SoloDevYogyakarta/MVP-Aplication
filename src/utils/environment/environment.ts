import { config } from 'dotenv';

config();

export const environment: NodeJS.ProcessEnv = process.env;
