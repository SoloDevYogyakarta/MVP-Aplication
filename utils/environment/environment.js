"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.environment = void 0;
const dotenv_1 = require("dotenv");
(0, dotenv_1.config)();
exports.environment = process.env;
//# sourceMappingURL=environment.js.map