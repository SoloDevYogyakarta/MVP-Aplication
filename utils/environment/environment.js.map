{"version":3,"file":"environment.js","sourceRoot":"","sources":["../../../src/utils/environment/environment.ts"],"names":[],"mappings":";;;AAAA,mCAAgC;AAEhC,IAAA,eAAM,GAAE,CAAC;AAEI,QAAA,WAAW,GAAsB,OAAO,CAAC,GAAG,CAAC"}