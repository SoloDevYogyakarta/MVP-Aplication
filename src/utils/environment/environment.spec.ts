import { environment } from './environment';

describe('environment', () => {
  it('should to be defined', () => expect(environment).toBeDefined());

  it('render correctly', () => expect(environment).toMatchSnapshot());

  it('read DB_HOST', () => expect(environment['DB_HOST']).toEqual('127.0.0.1'));
});
