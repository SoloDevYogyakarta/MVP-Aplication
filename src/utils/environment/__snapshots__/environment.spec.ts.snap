// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`environment render correctly 1`] = `
{
  "ANDROID_HOME": "/Users/kenedy-/Library/Android/sdk",
  "CLOUD_API_ACCESS_TOKEN": "EAA0H2i5l7sABOw3aS78bJ4lwO07XpWNblJLB9ae1TSBKXSdAlyZAEQFERWDqymfN3DuBjjCPw9FgGl8TaoDtdC3H9CEnbCILSnVZB8E6UMHuqO2A8whHwwECMcyG02WkTSYXlCiM14vIKU8dMsqv9zr5J9kao7vqMutCChNdRmDk9vaoroJwcPJaoeZAZBmeEYoMmmxk1miGoSLSPgZDZD",
  "CLOUD_API_VERSION": "V16.0",
  "COLOR": "1",
  "COLORTERM": "truecolor",
  "COMMAND_MODE": "unix2003",
  "CPATH": "/usr/local/include",
  "DB_DIALECT": "sqlite",
  "DB_HOST": "127.0.0.1",
  "DB_NAME": "mvpapplication",
  "DB_PASS": "password",
  "DB_PORT": "5432",
  "DB_USER": "root",
  "EDITOR": "vi",
  "FORCE_COLOR": "1",
  "GEM_HOME": "/Users/kenedy-/.rvm/gems/ruby-3.0.0",
  "GEM_PATH": "/Users/kenedy-/.rvm/gems/ruby-3.0.0:/Users/kenedy-/.rvm/gems/ruby-3.0.0@global",
  "GIT_EXEC_PATH": "/Library/Developer/CommandLineTools/usr/libexec/git-core",
  "GIT_PREFIX": "",
  "HOME": "/Users/kenedy-",
  "HOMEBREW_CELLAR": "/opt/homebrew/Cellar",
  "HOMEBREW_PREFIX": "/opt/homebrew",
  "HOMEBREW_REPOSITORY": "/opt/homebrew",
  "INFOPATH": "/opt/homebrew/share/info:",
  "INIT_CWD": "/Users/kenedy-/projects/mvpapplication",
  "IRBRC": "/Users/kenedy-/.rvm/rubies/ruby-3.0.0/.irbrc",
  "JAVA_HOME": "/Library/Java/JavaVirtualMachines/jdk-17.jdk/Contents/Home",
  "JEST_WORKER_ID": "7",
  "LANG": "en_US.UTF-8",
  "LESS": "-R",
  "LIBRARY_PATH": "/usr/local/lib",
  "LISTENED_PORT": "3000",
  "LOGNAME": "kenedy-",
  "LSCOLORS": "Gxfxcxdxbxegedabagacad",
  "LS_COLORS": "di=1;36:ln=35:so=32:pi=33:ex=31:bd=34;46:cd=34;43:su=30;41:sg=30;46:tw=30;42:ow=30;43",
  "MANPATH": "/Users/kenedy-/.nvm/versions/node/v18.12.1/share/man:/opt/homebrew/share/man::",
  "MY_RUBY_HOME": "/Users/kenedy-/.rvm/rubies/ruby-3.0.0",
  "MallocNanoZone": "0",
  "NODE": "/Users/kenedy-/.nvm/versions/node/v18.12.1/bin/node",
  "NODE_ENV": "test",
  "NVM_BIN": "/Users/kenedy-/.nvm/versions/node/v18.12.1/bin",
  "NVM_CD_FLAGS": "-q",
  "NVM_DIR": "/Users/kenedy-/.nvm",
  "NVM_INC": "/Users/kenedy-/.nvm/versions/node/v18.12.1/include/node",
  "ORIGINAL_XDG_CURRENT_DESKTOP": "undefined",
  "PAGER": "less",
  "PATH": "/Users/kenedy-/projects/mvpapplication/node_modules/.bin:/Users/kenedy-/projects/mvpapplication/node_modules/.bin:/Users/kenedy-/projects/node_modules/.bin:/Users/kenedy-/node_modules/.bin:/Users/node_modules/.bin:/node_modules/.bin:/Users/kenedy-/.nvm/versions/node/v18.12.1/lib/node_modules/npm/node_modules/@npmcli/run-script/lib/node-gyp-bin:/Users/kenedy-/projects/mvpapplication/node_modules/.bin:/Users/kenedy-/projects/node_modules/.bin:/Users/kenedy-/node_modules/.bin:/Users/node_modules/.bin:/node_modules/.bin:/Users/kenedy-/.nvm/versions/node/v18.12.1/lib/node_modules/npm/node_modules/@npmcli/run-script/lib/node-gyp-bin:/Library/Developer/CommandLineTools/usr/libexec/git-core:/Users/kenedy-/.rvm/gems/ruby-3.0.0/bin:/Users/kenedy-/.rvm/gems/ruby-3.0.0@global/bin:/Users/kenedy-/.rvm/rubies/ruby-3.0.0/bin:/Users/kenedy-/bin:/Users/kenedy-/.nvm/versions/node/v18.12.1/bin:/Library/Frameworks/Python.framework/Versions/3.7/bin:/Users/kenedy-~/.cargo/bin:/opt/homebrew/opt/php@7.3/sbin:/opt/homebrew/opt/php@7.3/bin:/Users/kenedy-/.nvm/versions/node/v18.12.1/bin:/Library/Frameworks/Python.framework/Versions/3.12/bin:/Library/Frameworks/Python.framework/Versions/2.7/bin:/Library/Frameworks/Python.framework/Versions/3.7/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/kenedy-/.cargo/bin:/Users/kenedy-/Library/Android/sdk/emulator:/Users/kenedy-/Library/Android/sdk/platform-tools:/Users/kenedy-/.rvm/bin",
  "PWD": "/Users/kenedy-/projects/mvpapplication",
  "RUBY_VERSION": "ruby-3.0.0",
  "SDKROOT": "/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk",
  "SECRET": "secret",
  "SHELL": "/bin/zsh",
  "SHLVL": "5",
  "SSH_AUTH_SOCK": "/private/tmp/com.apple.launchd.kDwIKIZgBF/Listeners",
  "TERM": "xterm-256color",
  "TERM_PROGRAM": "Hyper",
  "TERM_PROGRAM_VERSION": "3.4.1",
  "TMPDIR": "/var/folders/1b/73d1rld14b1bf0ks3vc8d0cw0000gn/T/",
  "TS_JEST": "1",
  "USER": "kenedy-",
  "WA_PHONE_NUMBER_ID": "082228067652",
  "WEBHOOK_ENDPOINT": "webhook/",
  "WEBHOOK_VERFICATION_TOKEN": "1234567890abcd",
  "XPC_FLAGS": "0x0",
  "XPC_SERVICE_NAME": "0",
  "ZSH": "/Users/kenedy-/.oh-my-zsh",
  "_": "/Users/kenedy-/projects/mvpapplication/node_modules/.bin/jest",
  "__CFBundleIdentifier": "co.zeit.hyper",
  "__CF_USER_TEXT_ENCODING": "0x1F5:0x0:0x2",
  "husky_skip_init": "1",
  "npm_command": "exec",
  "npm_config_cache": "/Users/kenedy-/.npm",
  "npm_config_global_prefix": "/Users/kenedy-/.nvm/versions/node/v18.12.1",
  "npm_config_globalconfig": "/Users/kenedy-/.nvm/versions/node/v18.12.1/etc/npmrc",
  "npm_config_init_module": "/Users/kenedy-/.npm-init.js",
  "npm_config_legacy_peer_deps": "true",
  "npm_config_local_prefix": "/Users/kenedy-/projects/mvpapplication",
  "npm_config_metrics_registry": "https://registry.npmjs.org/",
  "npm_config_node_gyp": "/Users/kenedy-/.nvm/versions/node/v18.12.1/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js",
  "npm_config_noproxy": "",
  "npm_config_prefix": "/Users/kenedy-/.nvm/versions/node/v18.12.1",
  "npm_config_user_agent": "npm/8.19.2 node/v18.12.1 darwin arm64 workspaces/false",
  "npm_config_userconfig": "/Users/kenedy-/.npmrc",
  "npm_execpath": "/Users/kenedy-/.nvm/versions/node/v18.12.1/lib/node_modules/npm/bin/npx-cli.js",
  "npm_lifecycle_event": "npx",
  "npm_lifecycle_script": "jest",
  "npm_node_execpath": "/Users/kenedy-/.nvm/versions/node/v18.12.1/bin/node",
  "npm_package_config_commitizen_path": "git-cz",
  "npm_package_json": "/Users/kenedy-/projects/mvpapplication/package.json",
  "npm_package_name": "mvp",
  "npm_package_version": "0.0.1",
  "rvm_alias_expanded": "",
  "rvm_bin_flag": "",
  "rvm_bin_path": "/Users/kenedy-/.rvm/bin",
  "rvm_docs_type": "",
  "rvm_gemstone_package_file": "",
  "rvm_gemstone_url": "",
  "rvm_hook": "",
  "rvm_niceness": "",
  "rvm_nightly_flag": "",
  "rvm_only_path_flag": "",
  "rvm_path": "/Users/kenedy-/.rvm",
  "rvm_prefix": "/Users/kenedy-",
  "rvm_pretty_print_flag": "",
  "rvm_proxy": "",
  "rvm_quiet_flag": "",
  "rvm_ruby_bits": "",
  "rvm_ruby_file": "",
  "rvm_ruby_make": "",
  "rvm_ruby_make_install": "",
  "rvm_ruby_mode": "",
  "rvm_script_name": "",
  "rvm_sdk": "",
  "rvm_silent_flag": "",
  "rvm_use_flag": "",
  "rvm_version": "1.29.12 (latest)",
}
`;
